bodhi/server/buildsys.py

```diff
@@ -78,6 +78,8 @@ class DevBuildsys(Buildsystem):
     __added__ = []
     __tagged__ = {}
     __rpms__ = []
+    multicall = False
+    multicallresult = []
```

Comment:
Not intentional at all. Just me not really knowing the difference there. Thanks for the lesson!

I have changed this now.